updates chart for new universal redis client

asoorm opened the pull request:
resolves #68

vesko-tyk commented on values.yaml:
```diff
@@ -89,19 +97,19 @@ tib:
   replicaCount: 1
   hostName: "localhost"
-  license: ""
+  license: "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhbGxvd2VkX25vZGVzIjoiOGU4NWFjZTUtNGRjOC00OTk1LTRkNzUtNWI4YTM2YzA4ZTJmLDViMDAzYzI1LTg3ZjYtNDIyZC00ODBhLWRiMGNjZDdhNjBlYyw2NWM4NDhhMy0xYWU4LTRhNzQtNGYwOS05ZjA5YmNkYjM5MGQsZmIwMTZhM2EtZjAxYi00ZWJiLTRlYzQtODI0NjkzYzE0ZWJmLDBkMDYwYTQ3LWUwMTktNDU1My01MGFhLWVjOWY4ZTEzZGE3YiwxNzdjMmZiZC04ZjRiLTQxZjEtNTg2ZC01MGE5YjQ0M2UzZWQsYjZjNGVjN2QtZGMwMi00NTg4LTcyMG ...
```
@asoorm Could you remove this please? The license should not be exposed here. Thanks

excieve reviewed (changes requested):
Nice, thanks! I'd suggest using a YAML list where variable is logically always a list of items for better usability.

drpebcak commented on values.yaml:
```diff
@@ -12,13 +12,21 @@ secrets:
     shardCount: 128
-    host: "tyk-redis-master.tyk-ingress.svc.cluster.local"
-    port: 6379
+    addrs: "tyk-redis-master.tyk-ingress.svc.cluster.local:6379"
```
I agree the list is easier, if you are using values.yaml. If you are using `--set`, it becomes a pain to access elements of the list.

drpebcak replied:
(personally I use values.yaml, so I'm all for that)

joshblakeley replied:
Thanks Taylor - i'm willing to ignore the --set crowd also

drpebcak commented:
Any movement on this?

asoorm commented:
> Any movement on this?

Hello - I will try to make changes today.

asoorm commented:
@excieve PTAL

asoorm commented:
After further testing, I found a couple of issues with erroneous log messages:

```
time="Nov 08 08:52:25" level=error msg="Error trying to get TTL: MOVED 609 10.244.0.60:6379"
time="Nov 08 08:52:26" level=error msg="Redis health check failed" error="storage: Redis is either down or ws not configured" liveness-check=true prefix=main
time="Nov 08 08:52:34" level=error msg="Error trying to get TTL: MOVED 609 10.244.0.60:6379"
time="Nov 08 08:52:36" level=error msg="Redis health check failed" error="storage: Redis is either down or ws not configured" liveness-check=true prefix=main
time="Nov 08 08:52:40" level=error msg="Error trying to get TTL: MOVED 609 10.244.0.60:6379"
time="Nov 08 08:52:40" level=error msg="Could not EXPIRE key: MOVED 609 10.244.0.60:6379"
```

Moving to draft till I figure this out.

vesko-tyk commented:
This is now implemented in this PR:https://github.com/TykTechnologies/tyk-helm-chart/pull/73
